package.json

```diff
@@ -46,6 +46,7 @@
     "pretty-error": "2.1.1",
     "raw-loader": "0.5.1",
     "sass-loader": "7.1.0",
+    "svg-inline-loader": "^0.8.0",
```

Comment:
Probably best to stay consistent and remove the caret

```suggestion
    "svg-inline-loader": "0.8.0",
```

Reply:
Nach entfernen nochmal die yarn.lock erstellen lassen.
Sonst super.

Reply:
Done